docs/installation.rst

```diff
@@ -1,24 +1,65 @@
-#################
-Installing JDAViz
-#################
+Installation
+============
+
+The following details how to install the ``jdaviz`` Python package.
+
+Quick Installation
+------------------
 
 Installing can be done using pip::
 
+    pip install --upgrade jdaviz
+
+or via Github::
+
    pip install git+https://github.com/spacetelescope/jdaviz --upgrade
 
-Or from source::
+Developer Installation
+----------------------
+
+To install ``jdaviz`` for development, or from source::
 
    git clone https://github.com/spacetelescope/jdaviz.git
    cd jdaviz
    pip install -e .
 
-Once installed, to start the browser app version, run the following in your terminal::
 
-    jdaviz /path/to/data/file --layout=cubeviz
+Running Jdaviz
+==============
+
+Once installed, ``jdaviz`` can be run either as a desktop application or in a Jupyter notebook.
+
+As a Desktop Application
+------------------------
+
+``jdaviz`` provides a command-line tool to start the desktop application.  To see the syntax and usage,
+from a terminal, type::
+
+    jdaviz --help
 
-For the notebook version, start the jupyter kernel with the path to the Example.ipynb notebook::
+The general syntax to start the application is to provide a local filename path and an application configuration
+to load, i.e.::
+
+    jdaviz /path/to/data/file --layout=[configuration]
+
+For example, to load a SDSS MaNGA IFU data cube into ``CubeViz``, you would run the following from a terminal::
+
+    jdaviz /my/manga/cube/manga-8485-1901-LOGCUBE.fits.gz --layout=cubeviz
+
+In a Juypter Notebook
+---------------------
+
+``jdaviz`` provides a directory of sample notebooks to test the application, located in the `notebooks` sub-directory
+of the git repository.  `Example.ipynb` is provided as an example that loads a SDSS MaNGA IFU data cube with the
+``CubeViz`` configuration.  To run the provided example, start the jupyter kernel with the notebook path::
 
     jupyter notebook /path/to/jdaviz/notebooks/Example.ipynb
 
-Note that to load data in the notebook version, uncomment the cell in the notebook and
-provide the path to your data.
+or simply start a new Jupyter notebook and run the following in a cell::
+
+    from jdaviz import Application
+
+    app = Application()
+    app
+
+To learn more about the various ``jdaviz`` application configurations and loading data, see xxxx.
```

Comment:
Where does this intend on pointing to?

Reply:
I was intending to point this to whatever sort of Getting Started page got created, or an intro page for CubeViz, SpecViz, etc.  If we have a placeholder reference, I can pop that in.